packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.test.tsx

```diff
@@ -51,13 +52,9 @@ describe('UnitRowTwoStepAuth', () => {
     expect(
       screen.getByTestId('two-step-unit-row-header-value').textContent
     ).toContain('Disabled');
-    expect(screen.getByTestId('two-step-unit-row-route').textContent).toContain(
-      'Add'
-    );
-    expect(screen.getByTestId('two-step-unit-row-route')).toHaveAttribute(
```

Comment:
This glean click event was unnecessary duplication of the custom event that is tracked in Looker dashboards. (Thanks for spotting, @MagentaManifold !)